scania/corporate-ui-site
# Improvement - landing page

**Describe pull-request**  
Landing page added.

**How to test**  
- See figma >> Landing page
- See in different breakpoints
